src/main/java/com/icthh/xm/tmf/ms/resource/config/DatabaseConfiguration.java

```diff
@@ -13,6 +12,8 @@
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 import org.springframework.transaction.annotation.EnableTransactionManagement;
 
+import java.sql.SQLException;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version